docs/sdk/web3-react/guides/01-connect-wallet.md

```diff
@@ -1,4 +1,165 @@
 ---
 id: connect-wallet
 title: Connecting to Wallets
----     
\ No newline at end of file
+---     
+
+## Introduction
+
+This guide will cover how to connect wallets with `web3-react`. It is based on the [web3-react example](https://github.com/Uniswap/examples/tree/main/web3-react), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples). To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/web3-react/README.md) and follow the setup instructions.
+
+
+In this example we will walk through setting up `web3-react` and connecting the most popular browser injected connector, *MetaMask*, using the web3-react [MetaMask package](https://www.npmjs.com/package/@web3-react/metamask).
+
+
+The input parameters to this guide are the chains that we want our app to be able to connect to and their RPC URLs.
+
+The guide will **cover**:
+
+1. Creating a `Web3ReactProvider`
```

Comment:
I think this should be briefly defined here, where it is first introduced:

    1. Creating a web3-react React `Context.Provider`, which we'll call `Web3ReactProvider`

Reply:
Also, isn't this called `Web3ContextProvider`? `Web3ReactProvider` is the component exported by `@web3-react`.

Reply:
We intentionally keep the titles short, but I can definitely elaborate later.